python/constellation/satellites/LeCroy/meson.build

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2024 DESY and the Constellation authors
+# SPDX-License-Identifier: CC0-1.0
```

Comment:
@stephanlachnit do we need this? Dependency installation should go through pyproject.toml now